test/DynamoCoreTests/Models/PresetCommandTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Linq;
+using System.Xml;
+using Dynamo.Graph.Nodes;
+using Dynamo.Models;
+using Dynamo.Selection;
+using NUnit.Framework;
+
+namespace Dynamo.Tests.ModelsTest
+{
+    /// <summary>
+    /// This test class will contain test cases for executing methods in the AddPresetCommand and ApplyPresetCommand clases
+    /// </summary>
+    [TestFixture]
+    class PresetCommandTests : DynamoModelTestBase
+    {
+        /// <summary>
+        /// This test method will execute the next method from the AddPresetCommand class
+        /// public AddPresetCommand(string name, string description, IEnumerable<Guid> currentSelectionIds )
+        /// static AddPresetCommand DeserializeCore(XmlElement element)
+        /// </summary>
+        [Test]
+        [Category("UnitTests")]
+        public void AddPresetCommand_DeserializeCore()
+        {
+            //Arrange
+            var ids = DynamoSelection.Instance.Selection.OfType<NodeModel>().Select(x => x.GUID).ToList();
+            var AddPresetCommand = new DynamoModel.AddPresetCommand("PresetName", "Description", ids);
+            XmlDocument xmlDocument = new XmlDocument();        
+            XmlElement elemTest = xmlDocument.CreateElement("TestCommand");
+
+            //Act//Assert
```

Comment:
Copy paste error here?

Reply:
@QilongTang I've fixed this comment and in other 2 test methods.

Please let me know if you have any other comment.

commit:
https://github.com/DynamoDS/Dynamo/pull/10640/commits/d5f1a2d07e79a4e251949d8f85b953bb8d8cbda6